tests/bearlib/languages/documentation/DocumentationExtractionTest.py

```diff
@@ -180,8 +180,15 @@ def test_extract_documentation_PYTHON3(self):
                          'short'),
                         docstyle_PYTHON3_default, '',
                         docstyle_PYTHON3_default.markers[0],
-                        TextRange.from_values(40, 1, 42, 9)))
-
+                        TextRange.from_values(40, 1, 42, 9)),
+                    DocumentationComment(
+                        ('\n'
+                         'A bad indented docstring\n'
+                         '    Improper indentation.\n'
+                         ':param impact: The force of Impact.\n'),
+                        docstyle_PYTHON3_default, '',
+                        docstyle_PYTHON3_default.markers[0],
+                        TextRange.from_values(45, 5, 49, 8)))
```

Comment:
Add a trailing `,` (and move the closing `)` to the next line, so you can add another one later without modifying this line.